inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/render/PdfAnnoRenderer.java

```diff
@@ -0,0 +1,281 @@
+/*
```

Comment:
Hm... why does this file appear as deleted and re-added instead of being moved?

Reply:
I don't know, was wondering about this as well.
Maybe because I stashed the changes in between or so?

Reply:
Can I fix this somehow?